recipes-docs/rzg2l-readme/files/README.md

```diff
@@ -0,0 +1,1845 @@
+│               ├── tools
+│               │   ├── cygterm.cfg
+│               │   ├── TERATERM.INI
+│               │   ├── ttermpro.exe
+│               │   ├── ttpcmn.dll
+│               │   ├── ttpfile.dll
+│               │   ├── ttpmacro.exe
+│               │   ├── ttpset.dll
+│               │   ├── ttxssh.dll
+│               │   └── uload-flash_bootloader.ttl
+│               └── uload-flash_bootloader.bat                           <---- Bootloader flashing from U-Boot console script on Windows
+├── license
+│   ├── Disclaimer051.pdf
+│   └── Disclaimer052.pdf
+├── r12uz0158eu0101-rz-g2l-sbc-single-board-computer.pdf
+├── README.md                                                            <---- This document
+├── RZG2L-SBC_Evaluation_license.pdf
+└── target                                                               <---- Holds images for bootloader, kernel, rootfs, and device tree
+    ├── env                                                              <---- Contains environment configuration files for booting and system setup
+    │   ├── Readme.md
+    │   └── uEnv.txt
+    ├── images                                                           <---- Contains bootloader, kernel, and root filesystem images
+    │   ├── bl2_bp-rzg2l-sbc.bin
+    │   ├── bl2_bp-rzg2l-sbc.srec
+    │   ├── bl2-rzg2l-sbc.bin
+    │   ├── ${IMAGE_BASENAME}-rzg2l-sbc.wic
+    │   ├── dtbs                                                         <---- Contains Device tree blobs (DTBs) for hardware configuration
+    │   │   ├── overlays                                                 <---- Overlays for extending device tree functionality
+    │   │   │   ├── Readme.md
+    │   │   │   ├── rzg2l-sbc-can.dtbo
+    │   │   │   ├── rzg2l-sbc-dsi.dtbo
+    │   │   │   ├── rzg2l-sbc-ext-i2c.dtbo
+    │   │   │   ├── rzg2l-sbc-ext-spi.dtbo
+    │   │   │   └── rzg2l-sbc-ov5640.dtbo
+    │   │   ├── Readme.md
+    │   │   ├── rzpi--5.10.184-cip36+gitAUTOINC+5f065ec41b-r1-rzg2l-sbc-20240910054534.dtb
+    │   │   └── rzpi.dtb -> rzpi--5.10.184-cip36+gitAUTOINC+5f065ec41b-r1-rzg2l-sbc-20240910054534.dtb
+    │   ├── fip-rzg2l-sbc.bin
+    │   ├── fip-rzg2l-sbc.srec
+    │   ├── Flash_Writer_SCIF_rzg2l-sbc.mot
+    │   ├── Image -> Image--5.10.184-cip36+gitAUTOINC+5f065ec41b-r1-rzg2l-sbc-20240910054534.bin
+    │   ├── Image--5.10.184-cip36+gitAUTOINC+5f065ec41b-r1-rzg2l-sbc-20240910054534.bin
+    │   ├── Readme.md
+    │   └── rootfs                                                       <---- Contains compressed root filesystem images
+    │       ├── ${IMAGE_BASENAME}-rzg2l-sbc.tar.bz2
+    │       └── Readme.md
+    └── Readme.md
+
+28 directories, 92 files
+```
+
+**The above structure is an example when building using the target image `IMAGE=${IMAGE_BASENAME}`. The compressed root filesystems and the environment artifacts will have names with the prefix `${IMAGE_BASENAME}`. Other target images will have the same structure.**
```

Comment:
Also, this IMAGE_BASENAME here and other in this README file, it's too ambiguous.